src/Components/RecentJobs/RecentBox/SingleJob/Skill/Skill.jsx

```diff
@@ -0,0 +1,11 @@
+function Skill({skill}) {
+
+return (
+
+    <button className="rounded bg-cyan-500 text-white m-1">{skill.skill}</button>
```

Comment:
You shouldn't really use a button for this, because the skills in this section aren't interactive yet, using a button tag creates accessiblity issues. 

Another option if changing it from a button is going to be painful is to add the disabled attribute to the button tag https://www.w3schools.com/tags/att_button_disabled.asp